types/components/meshes/Dodecahedron.d.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  MeshComponent,
+  MeshComponentParams
+} from '../../core/MeshComponent';
+
+import {
+  BufferGeometry,
+  DodecahedronGeometry,
+  Geometry,
+  Mesh
+} from 'three';
+
+/**
+ * this is missing in @types from three.js :(
```

Comment:
best would be to fix `@types/three` imo.
Unless there is a reason they don't provide interfaces for certain buffer geometries...